ui-grid/ui-grid-tests.ts

```diff
@@ -100,11 +108,25 @@ gridApi.core.queueGridRefresh()
 gridApi.core.queueRefresh();
 gridApi.core.registerColumnsProcessor(colProcessor, 100);
 
-var rowEntityToScrollTo = {anObject: 'inGridOptionsData'};
+var gridOptions: uiGrid.IGridOptionsOf<IMyEntity> = {
+    data: [{name: 'Bob', age: 100}],
+    onRegisterApi: (api) => {
+        api.selection.on.rowSelectionChanged(null, (row) => {
+            console.log(row.entity.name);
+        })
+    }
+}
+interface IAnotherEntity {
+    anObject: string
+}
+var anotherGridInstance: uiGrid.IGridInstance;
+var rowEntityToScrollTo = {
+    anObject: 'inGridOptionsData'
+};
 var columnDefToScrollTo: uiGrid.IColumnDef;
-gridInstance.scrollTo();
-gridInstance.scrollTo(rowEntityToScrollTo);
-gridInstance.scrollTo(rowEntityToScrollTo, columnDefToScrollTo);
+anotherGridInstance.scrollTo();
+anotherGridInstance.scrollTo(rowEntityToScrollTo);
+anotherGridInstance.scrollTo(rowEntityToScrollTo, columnDefToScrollTo);
 
 var selectedRowEntities: Array<any> = gridApi.selection.getSelectedRows();
 var selectedGridRows: Array<uiGrid.IGridRow> = gridApi.selection.getSelectedGridRows();
```

Comment:
And this be `var selectedGridRows: Array<uiGrid.IGridRowOf<IMyEntity>> = gridApi.selection.getSelectedGridRows();` ?